functions.php

```diff
@@ -88,6 +88,8 @@ function penciletto_content_width() {
  * Penciletto functions and definitions.
  */
 function penciletto_setup() {
+	add_theme_support( 'automatic-feed-links' );
+	add_theme_support( 'title-tag' );
```

Comment:
@DivyanshKhanna26 not sure if this is needed, as I believe it's added in parent theme. What do you think? Or theme chack is complaining about it?